packages/lit/rollup.config.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {litProdConfig} from '../../rollup-common.js';
+
+export default litProdConfig({
+  classPropertyPrefix: 'Φ',
+  entryPoints: [
```

Comment:
Do we need a rollup module build for `lit` at all? There should be no code here, and every re-export is to an out-of-package module. Is there any benefit? Would be nice to avoid that if possible.

If we need keep it for some reason, `classPropertyPrefix` should at least be something different than `lit-element`'s, to protect against some class source winding up in this module for some reason.

Reply:
We could probably get away without most of the rollup build, but copying the standard setup we have for other projects is a really easy way to get whatever small minification we can and copy the files to their final location. I changed the classPropertyPrefix, but it's not used now. We do need some rollup step, or a copy step, to get the platform-support.js file.

I suppose we can change the TS config to not output to `/development/`. otoh, that's a transparent change too...